ccmlib/cmds/command.py

```diff
@@ -79,7 +79,7 @@ def _get_default_parser(self, usage, description, ignore_unknown_options=False):
         else:
             parser = OptionParser(usage=usage, description=description)
         parser.add_option('--config-dir', type="string", dest="config_dir",
-            help="Directory for the cluster files [default to ~/.ccm]")
+            help="Directory for the cluster files [default to %s]" % common.get_default_path_display_name())
```

Comment:
This method of string formatting isn't Python-3 compliant. Can you use `.format()` instead?